client/src/components/auth/registerFrom.tsx

```diff
@@ -1,75 +1,82 @@
 import * as React from "react";
 
 export default function RegisterForm() {
-  const [mail, setMail] = React.useState("");
-  const [username, setUsername] = React.useState("");
-  const userInfo = { mail: mail, username: username };
+    const [mail, setMail] = React.useState("");
+    const [username, setUsername] = React.useState("");
+    const userInfo = { mail: mail, username: username };
```

Comment:
I would put this in `handleRegister`, since you only use it there. No need to keep it in the component.

Reply:
How can I move this inside the function without react complaning about it neededing to be in a component?